src/PKSim.Core/Snapshots/Mappers/SimulationMapper.cs

```diff
@@ -149,7 +141,9 @@ private string[] usedObervedDataFrom(ModelSimulation simulation, PKSimProject pr
 
          await runSimulation(snapshot, simulation);
 
-         simulation.AddAnalyses(await simulationChartFrom(simulation, snapshot.Charts, project));
+         simulation.AddAnalyses(await individualAnalysesFrom(simulation, snapshot.IndividualAnalyses, project));
+//         simulation.AddAnalyses(await populationAnalysesFrom(simulation, snapshot.PopulationAnalyses, project));
```

Comment:
Is that not required anymore?

Reply:
oh, I see, it will be... later

Reply:
yes but this will be done in the import (this is about export here)